pyfolio/tears.py

```diff
@@ -669,6 +680,86 @@ def create_interesting_times_tear_sheet(
 
 
 @plotting_context
+def create_capacity_tear_sheet(returns, positions, transactions, 
+                               market_data,
+                               daily_vol_limit=0.2, last_n_days=125):
+    """
+    Generates a report detailing portfolio size constraints set by
+    least liquid tickers. Plots a "capacity sweep," a curve describing
+    projected sharpe ratio given the slippage penalties that are
+    applied at various capital bases.
+
+    Parameters
+    ----------
+    returns : pd.Series
+        Daily returns of the strategy, noncumulative.
+         - See full explanation in create_full_tear_sheet.
+    positions : pd.DataFrame
+        Daily net position values.
+         - See full explanation in create_full_tear_sheet.
+    transactions : pd.DataFrame
+        Prices and amounts of executed trades. One row per trade.
+         - See full explanation in create_full_tear_sheet.
+    market_data : pd.Panel
+        Contains OHLCV DataFrames for the tickers in the passed
+        positions DataFrames
+    daily_vol_limit : float
+        Max proportion of any daily bar that the startegy is allowed to
+        consume in the constraining ticker analysis.
+    last_n_days : integer
+        Compute max position allocation and dollar volume for only
+        the last N days of the backtest
+    """
+
+    whole_bt = capacity.get_assets_dollar_volume(positions,
+                                                 market_data,
+                                                 only_held_days=True)
+    n_days = capacity.get_assets_dollar_volume(positions,
+                                               market_data,
+                                               only_held_days=True,
+                                               last_n_days=last_n_days)
+    for title, data in [('Whole backtest', whole_bt),
+                        ('Last six months:', n_days)]:
+        print(title)
+        print('Most Liquid Names:')
+        utils.print_table(
+            data.sort('avg_daily_dv_$mm', ascending=False).head(5))
+        print('Least Liquid Names:')
+        utils.print_table(
+            data.sort('avg_daily_dv_$mm', ascending=True).head(8))
+
+        constraints = capacity.get_portfolio_size_constraints(
+            data, daily_vol_limit=daily_vol_limit)
+        constraining_tickers = capacity.get_constraining_tickers(constraints)
+        print("Max capacity at %s%% pct daily bar consumption limit" %
+              (daily_vol_limit * 100))
+        utils.print_table(constraining_tickers)
+
+    illiquid = whole_bt[whole_bt['avg_daily_dv_$mm'] < 1]
```

Comment:
Maybe also use a const here instead of `1` to make this more clear, like `MIN_DAILY_DV_MM`.